topics/go/algorithms/data/hash/hash.go

```diff
@@ -133,7 +133,8 @@ func (h *Hash) hashKey(key string) int {
 	h.hash.Reset()
 
 	// Write the key to the maphash to update the current state.
-	h.hash.WriteString(key)
+	// We don't check error value since WriteString never fails
```

Comment:
I don't understand this change in comment. It is not describing what we are doing. This comment could be provided along with the other one?  Don't forget proper sentence structure. Missing period.

Reply:
Wait, I see you added the second comment.

Again, this comment is confusing here, I need more. The extra nolint:errcheck needs to go as well.  People won't understand this.

Reply:
OK. I tried to fix all the linting error and `golangci-lint` complains about this line. Will remove the `nolint` comment. 
I can't think of better phrasing to `We don't check error value since WriteString never fails.`. Remove it?

Reply:
Just worry about `golint` working. That is from Google and it should be enough.